### DISCUSSION TITLE
cannot fork

### DISCUSSION THREAD

**Discussion by mwallr2d2** [community/community#54564] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Clearly I'm doing something wrong, just started with Github, could somebody advise regarding this prompt:

"cannot fork because you own this repository and are not a member of any organizations"

**Comment by GeoffreyFClark** [comment]
When you fork a repository, it creates a copy from a different user's account to your account.
You can't fork a repository that you already own on GitHub. 

I you want to copy your own repository could instead create a new repository and use the "import repository" feature or manually copy contents over. Alternatively you could create a new branch in the original repository or fork into an organization if you're part of one on GitHub.

**Reply by Jiayou-Chao** [reply]
I think this is quite different from a fork. You can't pull back and forth between the two repos.

**Comment by mwallr2d2** [comment]
Thank you very much for your response. Sent from my iPhoneOn May 5, 2023, at 4:24 PM, Geoffrey F. Clark ***@***.***> wrote:﻿
When you fork a repository, it creates a copy from a different user's account to your account.
You can't fork a repository that you already own on GitHub.
I you want to copy your own repository could instead create a new repository and use the "import repository" feature or manually copy contents over. Alternatively you could create a new branch in the original repository or fork into an organization if you're part of one on GitHub.

**Reply by GeoffreyFClark** [reply]
You're welcome. Feel free to mark the question as answered by selecting "mark as answer"
